packages/s/shapelib/monitoring.yml

```diff
@@ -0,0 +1,7 @@
+releases:
+  id: 4804
+  rss: ~
```

Comment:
Small nit: There is a github repo we can use: https://github.com/OSGeo/shapelib for a feed